testing/integration/ess/beat_receivers_test.go

```diff
@@ -872,13 +871,14 @@ agent.monitoring.enabled: false
 		status, statusErr := fixture.ExecStatus(ctx)
 		assert.NoError(collect, statusErr)
 		// we should be running beats processes even though the otel runtime was requested
-		assertBeatsHealthy(collect, &status, component.ProcessRuntimeManager, 1)
+		assertBeatsHealthy(collect, &status, component.ProcessRuntimeManager, 4)
```

Comment:
the 4 is the expected number of beats that are running? The change is due to enabling self-monitoring ?